test/multiverse/suites/agent_only/utilization_data_collection_test.rb

```diff
@@ -96,12 +96,12 @@ def redirect_link_local_address(port)
       @dummy_port = p
 
       class << self
-        def start_with_patch(address, port = nil, p_addr = nil, p_port = nil, p_user = nil, p_pass = nil, &block)
+        def start_with_patch(address, port, *_args, &block)
```

Comment:
were these arguments never used by whatever is calling this method?

Reply:
That's my understanding, yes. This method is defined in this test class and only used by the 2 test cases within - neither of which ever cared about the extra arguments. Without this change, the new Net::HTTP options hash was getting incorrectly mapped to the `p_addr` argument.

Reply:
that makes sense, thank you for explaining!